standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2502,6 +2520,9 @@ PartitionsResponse get_partitions_req(1:PartitionsRequest req)
                        throws(1:MetaException o1, 2:NoSuchObjectException o2)
   CheckConstraintsResponse get_check_constraints(1:CheckConstraintsRequest request)
                        throws(1:MetaException o1, 2:NoSuchObjectException o2)
+  // All table constrains
+  AllTableConstraintsResponse get_all_table_constraints(1:AllTableConstraintsRequest request)
```

Comment:
Here it shows, this method returns NoSuchObjectException but actually not. I think, the implementation should return this exception and handle in HMS client as well.

Reply:
I have add the NoSuchObjectException in get_all_table_constraints.